packages/web-frontend/src/components/View/MatrixWrapper/components/Cell.js

```diff
@@ -1,19 +1,86 @@
 import React from 'react';
+import { getPresentationOption } from '../../../../utils';
 
-export const Cell = ({
+const ROW_INFO_KEY = '$rowInfo';
+
+// Nested row values will be used in `DescriptionOverlay.js`
+const getNestedRowsValue = ({ childRows, category, cellKey }) => {
+  const rowsByCategory = childRows.filter(row => row.categoryId === category);
+  return rowsByCategory.map(data => `${data.description}: ${data[cellKey] ?? null}`).join('\\n');
+};
+
+const getPresentation = (presentationOptions, value, presentationConfigIfInRow) => {
+  const presentation = getPresentationOption(presentationOptions, value);
+  if (presentationConfigIfInRow) {
```

Comment:
Based on the existing [code and comment](https://github.com/beyondessential/tupaia/blob/dev/packages/web-frontend/src/components/View/MatrixWrapper/components/Row.js#L119):

```suggestion
  if (presentation && presentationConfigIfInRow) {
```